.github/workflows/testing.yml

```diff
@@ -10,11 +10,11 @@ jobs:
   linting:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
```

Comment:
You can enable `dependabot` for GitHub Actions, so all actions will be updated automatically.

Reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#supported-repositories-and-ecosystems

Reply:
Yeah true, I use this in a few other projects, and might as well add it here too :+1: